File: dumb.cs
using System;
using System.Diagnostics;
namespace dumb
{
    class Program
    {
        static void Main()
        {
            Stopwatch stopwatch = new Stopwatch();
            stopwatch.Start();
            for (long i = 1; i <= Math.Pow(2,32); i++)
            {
                if (i % 1000000 == 0)
                {
                    Console.WriteLine(i);
                }
            }
            stopwatch.Stop();
            Console.WriteLine(stopwatch.Elapsed.TotalMilliseconds.ToString() + "ms total");
        }
    }
}

File: Dumb.vb
Imports System
Imports System.Diagnostics
Module Dumb
    Sub Main()
        Dim stopwatch As Stopwatch = Stopwatch.StartNew()
        For i As Long = 0 To Math.Pow(2, 32)
            If (i Mod 1000000 = 0) Then
                Console.WriteLine(i)
            End If
        Next
        stopwatch.Stop()
        Console.WriteLine(stopwatch.Elapsed.TotalMilliseconds)
        Console.ReadLine()
    End Sub

End Module
